packages/keyring/src/keyring.ts

```diff
@@ -194,7 +195,9 @@ export class Keyring implements KeyringInstance {
     }
 
     const derived = type === 'ethereum'
-      ? hdEthereum(seed, derivePath.substring(1))
+      ? isHex(phrase, 256)
+        ? keypairFromSeed[type](seed) // for eth, if the private key is provided as suri, it must be derived only once
```

Comment:
In this case it means you cannot do derivations on the supplied key.

Reply:
It means that derivation will only be done for a mnemonic.
I don't understand what you are saying about the UI

Reply:
On the UI, you have this -

![image](https://user-images.githubusercontent.com/1424473/109511643-8a17b700-7aa3-11eb-8f9e-794abb49e816.png)

So in the case of the raw seeds, since the keyring has no support for derivation, it should not have the derivation option.

Reply:
ha yes indeed

Reply:
let me take care of the common bump when its ready and I will make the necessary changes